app/kuma-dp/pkg/dataplane/metrics/metrics_mutator.go

```diff
@@ -15,7 +16,7 @@ type (
 
 func AggregatedOtelMutator(metricsMutators ...PrometheusMutator) MeshMetricMutator {
 	return func(in io.Reader) (map[string]*io_prometheus_client.MetricFamily, error) {
-		var parser expfmt.TextParser
+		parser := expfmt.NewTextParser(model.LegacyValidation)
```

Comment:
Should we create issue to stop using legacyValidation?

Reply:
True, I am going to create